adafruit_emc2101/__init__.py

```diff
@@ -375,7 +349,7 @@ def manual_fan_speed(self):
         raw_setting = self._fan_setting & emc2101_regs.MAX_LUT_SPEED
         fan_speed = self._full_speed_lsb
         if fan_speed < 1:
-            raise EMC2101BadValueException()
+            raise RuntimeError("Connection")
```

Comment:
If this is also caused by faulty communication with the sensor, it should be `OSError`.